_authors/amitgalitz.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Amit Galitzky
+short_name: amitgalitz
+photo: ‘/assets/media/authors/amitgalitz.jpg’
```

Comment:
Please modify ‘/assets/media/authors/amitgalitz.jpg’ to '/assets/media/authors/amitgalitz.jpg' 
(it seems the ‘ was a different character set than ' and was causing the author picture to not display in Jekyll.)